scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java

```diff
@@ -90,7 +91,7 @@
     private Map<TaskId, EligibleTaskDescriptor> branchTasks = new ConcurrentHashMap<>();
 
     /** Job running tasks */
-    private Map<TaskId, TaskDescriptor> runningTasks = new ConcurrentHashMap<>();
+    public Map<TaskId, TaskDescriptor> runningTasks = new ConcurrentHashMap<>();
 
     /** Job paused tasks */
     private Map<TaskId, EligibleTaskDescriptor> pausedTasks = new HashMap<>();
```

Comment:
What about the @XmlTransient like at the eligibleTasks ?

This is not a ConcurrentHashMap like the others.